site/components/grid/index.js

```diff
@@ -35,10 +36,10 @@ type GridProps = {
 
 export class Grid extends React.Component<GridProps> {
   static propTypes = {
-    breakOn: React.PropTypes.string,
-    children: React.PropTypes.node,
-    extraClassName: React.PropTypes.string,
-    fit: React.PropTypes.bool,
+    breakOn: PropTypes.string,
+    children: PropTypes.node,
+    extraClassName: PropTypes.string,
+    fit: PropTypes.bool,
   };
```

Comment:
this propType is not needed, the exact same thing is covered by the flow type above